upup/pkg/fi/cloudup/openstacktasks/floatingip.go

```diff
@@ -171,11 +169,59 @@ func (e *FloatingIP) Find(c *fi.Context) (*FloatingIP, error) {
 			e.IP = actual.IP
 			return actual, nil
 		}
+
 	}
 
+	if len(fips) == 0 {
+		//If we fail to find an IP address we need to look for IP addresses attached to a port with similar name
+		//TODO: remove this in kops 1.21 where we can expect that the description field has been added
```

Comment:
But what will add it? does user manually add that? or should our code do that?

Reply:
The PR was missing the change to the RenderOpenstack function. Fixed that now. Hope that makes it more clear.